### ISSUE TITLE
Django 1.9 moved jQuery scripts

### ISSUE THREAD

**Issue by tongwang** [digi604/django-smart-selects#117]
Django 1.9 has moved jQuery scripts from `django/contrib/admin/static/admin/js` to `django/contrib/admin/static/admin/js/vendor/jquery`. If `USE_DJANGO_JQUERY` is set to `True`, will get HTTP 404 Not Found in Django 1.9:

https://github.com/digi604/django-smart-selects/blob/e0112d1c5313a78fa34312ae9d36751a86183e90/smart_selects/widgets.py#L51:L56

**Comment by SalahAdDin** [comment]
Yes, i have the same problem:
![seleccion_129](https://cloud.githubusercontent.com/assets/5034215/13031898/6fd1f72e-d2ac-11e5-8bab-5a921b9b7d0f.png)

I solve this putting:
```html
{% block extra_css %}
    <!-- TODO: Delete this if Smart-Selects solve this problem -->
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.min.js"></script>
{% endblock %}
```

But, isn't the best way!

**Comment by SalahAdDin** [comment]
Yes, adding vendor we haven't problems, but, now, the problem is using jQuery url.
I'll create new issue for this.

**Comment by millin** [comment]
Please look to my commit
#115

**Comment by SalahAdDin** [comment]
Ok, this solve when debug is false, but, when debud is true?

**Comment by millin** [comment]
@SalahAdDin, for me when Debug is True all works correct too, I'm just checked this.

**Comment by SalahAdDin** [comment]
Seriously? It doesn't works for me. Until now, the avoid that i have put `USE_DJANGO_JQUERY =FALSE`, but doesn't solve the other problems.

**Comment by jimmykobe1171** [comment]
@SalahAdDin when debug=True, does the problem still exist?  I can not reproduce in my computer, could you post some error trace stack? Thanks.

**Comment by SalahAdDin** [comment]
All my problems are in `debug=True`, i'm not in production yet.

**Comment by jimmykobe1171** [comment]
@SalahAdDin I mean the `ValueError running collectstatic`. The Jquery problem should be fixed. You can try on the branch master.

**Comment by SalahAdDin** [comment]
No, these no. I said, i'm working in developt for now, but, when i'll go production environment, i'll say you. :D

**Comment by blag** [comment]
Fixed in #137, which should be merged shortly. I'm closing this, but feel free to comment/reopen if necessary.
